assets/stylesheets/styles.css

```diff
@@ -0,0 +1,87 @@
+@import url('https://fonts.googleapis.com/css?family=Stint+Ultra+Expanded');
+@import url('https://fonts.googleapis.com/css?family=Pontano+Sans');
+
+/*General Styling*/
+body *{
+  margin: 0px;
+  padding: 0px;
+  border-width: 0px;
+}
+
+body{
+  font-family:'Pontano Sans',Helvetica,sans-serif;
+
+  background: #3a7bd5; /* fallback for old browsers */
+  background: -webkit-linear-gradient(to left, #3a7bd5 , #3a6073); /* Chrome 10-25, Safari 5.1-6 */
+  background: linear-gradient(to left, #3a7bd5 , #3a6073); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+
+}
+
+body a{
+  color: #B9C9D9;
+  text-decoration: none;
+}
+
+/*Header Styling*/
+/*Header animation*/
+ @keyframes fade-in {
+   0% {
+     opacity: 0;
+   }
+   100% {
+     opacity: 1;
+   }
+ }
+
+ header h2{
+   display:inline-block;
+   font-family: 'Stint Ultra Expanded',Helvetica, serif;
+   width: 50%;
+   color:gray;
+   font-style: italic;
+   font-size:30px;
+   animation-name: fade-in;
+   animation-duration: 10s;
+   animation-delay: -1s;
+   font-size: 50px;
+   font-family: "Stint Ultra Expanded", cursive;
+   text-shadow: 1px 1px 2px gray, 0 0 1em #3a7bd5, 0 0 0.2em teal;
+
+ }
+
+ header nav:first-of-type{
+   display: inline-block;
+   width: 39%;
+   text-align: right;
+
+ }
+header nav > ul{
+  display: inline-block;
+}
+
+nav li {
+display: inline-block;
+padding: .1em;
+}
+/*Splash Section Styling*/
+#splash {
+    height: 200px;
+    vertical-align: text-bottom;
+    text-align: center;
+    padding-top: 20%;
+    margin-top: 10%;
+    margin-left: 0px
+    margin-bottom: 30%;
+    background-image: url('/Users/tehutgetahun/ada/que/week6/hw/Static-Site/assets/photos/market.JPG');
```

Comment:
You've got an absolute path here, which includes your username! This means that your image won't load on anyone else's machine.

It's always better to use relative paths for this sort of thing. In this case it would be something like `assets/photos/market.JPG` (note there's no leading `/`)